lib/sda/service.go

```diff
@@ -295,7 +297,7 @@ in place of the standard reply. The Client.Send method will catch that and retur
 
 // Client for a service
 type Client struct {
-	private abstract.Secret
+	Private abstract.Secret
```

Comment:
Do we have a use-case where a user of a `Client` uses this `Private`,i.e. why putting `Private` as a exported/public field ? This morning I don't see any.